src/IssueViz.UnitTests/Editor/ErrorTagging/ErrorTaggerTests.cs

```diff
@@ -65,6 +65,38 @@ public void GetTags_FilterIsApplied_ExpectedTagsCreated()
             actual[1].Span.Span.Should().Be(primary2.Tag.Location.Span.Value.Span);
         }
 
+        [TestMethod]
+        public void GetTags_EmptySpansShouldBeFiltered()
+        {
+            var snapshot = CreateSnapshot(length: 50);
+
+            var inputSpans = CreateSpanCollectionSpanningWholeSnapshot(snapshot);
+
+            var primary1 = CreateTagSpanWithPrimaryLocation(snapshot, new Span(1, 5), message: "error message 1", ruleKey: "cpp:S5350");
+            var primary2 = CreateTagSpanWithPrimaryLocation(snapshot, new Span(10, 0), message: "error message 2", ruleKey: "cpp:emptyCompoundStatement");
```

Comment:
This is a very subtle change, I had to do diff compare with the test above to know what actually makes the location invalid.
I think it would help to extract it to a variable with an explicit name (i.e. `invalidSpan`) and assign it.

Also, the secondary locations seem irrelevant here.